fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientIdentifier.java

```diff
@@ -48,6 +48,9 @@
     @Column(name = "document_key", length = 1000)
     private String documentKey;
 
+    @Column(name = "document_issue_country", length = 1000)
```

Comment:
Let's be very strict with this. If the country code is 2-3 characters, let's stick with the minimum length instead of 1000. I think 3 should be good enough.